packages/backend/src/old/services/balances/utils/parseTokenPrices.ts

```diff
@@ -1,7 +1,7 @@
+import { divOrZero, tokenIsBefore } from '@l2beat/common'
```

Comment:
Keep this in backend we are going to replace tokenIsBefore with EthereumAddress.inOrder